### PR TITLE
New Pull Request (lvl 71-80 values) as requested

### PR DESCRIPTION
**Pull request by ShammyLevva**
Moved 71-80 values into new pull request
Added preparatory touch icon

### CODE REVIEW

**Comment by ShammyLevva** [comment]
This pull request adds the crafter level values and a first estimate of the polynomial for the calculation. 

The polynomial isn't perfect yet of course but it gives a fairly close approximation of the progress figures required post 70 crafting and thus makes the simulation usable for ShadowBringer crafting.

Further tweaks are required to improve of course but for now these changes make the app usable for the masses.

**Comment by doxxx** [comment]
Hi, thanks for the PR! Sorry I've taken so long to get to it, it's been a busy week.

File: app/js/ffxivcraftmodel.js [hunk 1]
```diff
@@ -101,9 +104,15 @@ Synth.prototype.calculateBaseProgressIncrease = function (levelDifference, craft
...
     if (levelDifference < 0) {
-        levelCorrectionFactor += 0.025 * Math.max(levelDifference, -10);
-        if (ProgressPenaltyTable[recipeLevel]) {
-            recipeLevelPenalty += ProgressPenaltyTable[recipeLevel];
+        if (crafterLevel > 380) {
+            // Level penalty for recipes above crafter level
+            levelCorrectionFactor += 0.025 * Math.max(levelDifference, -10);
+            if (ProgressPenaltyTable[recipeLevel]) {
+                recipeLevelPenalty += ProgressPenaltyTable[recipeLevel];
+            }
```
Comment (doxxx) [inline app/js/ffxivcraftmodel.js]: I think this progress penalty needs to be applied at lower crafter levels as well. I think it should appear after the if block.

**Comment by doxxx** [comment]
How much data did you use to derive the new polynomial?

**Comment by ShammyLevva** [comment]
> How much data did you use to derive the new polynomial?

Not nearly enough. 😂

It was about 20 data points. It’s not good enough for live but gives a decent starting point for testing and way better than the wildly out values on beta just now. 

My thought process is to do enough so you can get an initial working post 70 version as beta test (perhaps heavily warn its beta post 70) then tweak as more data is provided. I’ve been tweaking the inginuity and inginuity II values as well in my local copy but they are harder to derive.

File: app/js/ffxivcraftmodel.js (hunk 1 above)
Comment (ShammyLevva) [reply app/js/ffxivcraftmodel.js]: Fixed.

**Comment by doxxx** [comment]
Deployed to beta site.

**Pull request merged**